Add k-notification docs (#1218)
kytos/kytos: docs/developer/events.rst

        }
    }
    </script>

**k-notification**

=============== ====== ===========================================================
name            type   description                        
=============== ====== =========================================================== 

> setNotification object Defines the icon, title and description of the notification 

Reviewer: ```suggestion
setNotification object Define the icon, title and description of the notification 
```

=============== ====== =========================================================== 

To use the ``k-notification``, a notification object is necessary. Description

> and icon are optional - the only required parameter is the tittle. Usage

Reviewer: ```suggestion
and icon are optional - the only required parameter is the title. Usage

```

> To use the k-notification the first thing is to define the notification object.
> The description is not required, so is the icon. These parameters are defined
> by default, but the user can custom it as well. As follows the example:

Reviewer: ```suggestion
To use ``k-notification``, a notification object is necessary.
Description and icon are optional - the only required parameter
is the title. Usage example:
```

.. code-block:: html
    
    <template>
        ...
    </template>

    <script>
    /* All the javascript methods are optional */
    module.exports = {
        methods: { 
            // put your javascript methods here
        },
        mounted() {
            let notification = {
                title: 'Notification Title', 

>                 description: 'Notf. Description', 
>                 icon:'gear'

Reviewer: If the description can be longer than the title, why abbreviate it? ;)

```suggestion
                description: 'Notification Description',
                icon: 'gear'
```
